tests/cases/compiler/implicitAnyFromCircularInference.ts

```diff
@@ -39,7 +39,6 @@ interface A {
 function foo(x: A): string { return "abc"; }
 
 class C {
-    // Error expected

```

Comment:
this was outdated, no error is being reported here (and no error has to be reported here)